docs/docs/migration-guide.mdx

```diff
@@ -159,16 +191,16 @@ pipeline:
 In addition you need to add a [Rule](./rules.mdx) to activate the Two-Stage Fallback for
 messages with low NLU confidence.
 
-```yaml-rasa
+```yaml-rasa title="rules.yml"
```

Comment:
would you please be so kind and also add the files `title` to  the ones for migrating the `FormPolicy`?